monty_hall/monty_hall.py

```diff
@@ -0,0 +1,31 @@
+import random
+
+SWITCH = True
+goats = 0
+boats = 0
+runs = 1000000
+
+for _ in range(runs):
```

Comment:
cool :+1: there's a couple other things (like lists) that I'm going to show, but say that Ben is going to cover in more depth next week